Source/Diffable Collection/FetchedDiffableSection.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  FetchedDiffableSection.swift
+//  FetchedDataSource
+//
+//  Created by Jonathan Provo on 14/03/2022.
+//
+
+import Foundation
+
+public struct FetchedDiffableSection: Hashable {
+	// MARK: - Properties
+
+	public let identifier: String
+
+	// MARK: - Lifecycle
+
+	public init(identifier: String) {
+		self.identifier = identifier
+	}
+
+	// MARK: - Hashable
```

Comment:
Split of the bit below into an extension that implements `Hashable`, is nicer to have that bundled together. See for example https://github.com/appwise-labs/AppwiseCore/blob/master/Sources/Core/Identity.swift#L108-L118